src/cljs/athens/style.cljs

```diff
@@ -0,0 +1,31 @@
+(ns athens.style (:require  [garden.core :refer [css]]))
+
+(defn loading-css
+  []
+  (fn []
+    [:style (css
+      [:body {
+        :font-family "sans-serif"
+        :font-size "1.3rem"
+      }]
+    )]
+  )
+)
+
+(defn main-css
+  []
+  (fn []
+    [:style
+      (clojure.string/join "" [
+          (css [:body {:font-family "sans-serif"}])
+          (css :.pages-table [
+            :th {
+              :font-weight "bold"
+              :min-width "11em"
+            }
+          ])
+        ]
+      )
+    ]
+  )
+)
```

Comment:
We don't do this in clojure 😎 All brackets together. Most of us use an editor that can help us keeping track of the nesting. I'm adding a tool to check the styling so you can check these things beforehand